app/controllers/trips_controller.rb

```diff
@@ -77,6 +81,28 @@ def destroy
   def download
   end
 
+  def reject
+    if (can? :read, @trip) && @trip.status == 'applied'
+      @trip.update_attribute(:status, 'declined')
```

Comment:
Mit @trip.update_attributes(status: 'declined', last_modified: Date.today) gehts auch in einer Zeile :)